prep-css-grid-classes/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Grid Classes</title>
+  <link rel="stylesheet" href="reset.css">
+  <style>
+/* stylelint-disable block-no-empty */
+body {
+  font-family: sans-serif;
+  margin-bottom: 100px;
+}
+
+h1 {
+  text-align: center;
+}
+
+h3 {
+  height: 7%;
+}
+
+/* * {
+  border: 3px solid red;
+} */
+
+.container {
+  max-width: 992px;
+  margin: auto;
+}
+
+.row {
+  width: 100%;
+  display: flex;
+}
+
+.column {
+  flex-basis: calc(100%/3);
+  padding-left: 8px;
+  padding-right: 8px;
+}
+
+img {
+  width: auto;
+  height: auto;
+  object-fit: scale-down;
+  max-height: 400px;
+}
+        }
+
+  </style>
+</head>
+<body>
+
+<!-- Complete the HTML here -->
+
+</body>
+  <div class="container">
+    <h1>Indiana Jones</h1>
+    <div class="row">
+      <div class="column">
+        <img src="images/crusade.png" alt="Indiana Jones and the Last Crusade Poster">
+        <h3>Indiana Jones and the Last Crusade</h3>
+        <p>
+          This is the third installment in the Indiana Jones franchise and a sequel to Raiders of the Lost Ark. It is also the first Indiana Jones film to be rated PG-13. In the film, set largely in 1938, Indiana searches for his father, a Holy Grail scholar, who has been captured by Nazis.
+        </p>
+      </div>
+      <div class="column">
+        <img src="images/raiders.jpg" alt="Raiders of the Lost Ark Poseter">
+        <h3>Raiders of the Lost Ark</h3>
+        <p>
+          This is the first installment of the Indiana Jones franchise. Indiana Jones, a globe-trotting archaeologist vying with Nazi German forces in 1936 to recover the long-lost Ark of the Covenant, a relic said to make an army invincible. Teaming up with his tough former lover Marion Ravenwood, Jones races to stop rival archaeologist Dr. René Belloq from guiding the Nazis to the Ark and its power.
+        </p>
+      </div>
+      <div class="column">
+        <img src="images/temple.jpg" alt="Indiana Jones and the Temple of Doom Poster">
+        <h3>Indiana Jones and the Temple of Doom</h3>
+        <p>
+          This is the second installment in the Indiana Jones franchise, a prequel to the 1981 film Raiders of the Lost Ark. After arriving in India, Indiana Jones is asked by desperate villagers to find a mystical stone and rescue their children from a Thuggee cult practicing child slavery, black magic, and ritualistic human sacrifice in honor of the goddess Kali.
+        </p>
+      </div>
+    </div>
+  </div>
```

Comment:
Be sure to put your visible content inside of the `<body> </body>`. The browser is pretty forgiving and will try to display your page as best it can, but if the content is outside of the body, it'll confuse another developer.

Reply:
![div-to-body-fix](https://user-images.githubusercontent.com/82680001/121727075-44a25700-caa0-11eb-9e81-ae62391c75af.png)
Whoops! Totally overlooked that. It's fixed now. Thank you.